lading/src/observer/linux/cgroup/v2.rs

```diff
@@ -70,8 +72,21 @@ pub(crate) async fn poll(file_path: &Path, labels: &[(String, String)]) -> Resul
 
                                     match fs::read_to_string(&file_path).await {
                                         Ok(content) => {
-                                            let content = content.trim();
+                                            if file_name == "memory.pressure"
+                                                || file_name == "io.pressure"
+                                                || file_name == "cpu.pressure"
+                                            {
+                                                if let Err(err) =
+                                                    parse_pressure(&content, &metric_prefix, labels)
+                                                {
+                                                    debug!("[{path}] Failed to parse PSI contents: {err:?}",
```

Comment:
```suggestion
                                                    warn!("[{path}] Failed to parse PSI contents: {err:?}",
```

Lets find out early if we have a bug here

Reply:
Addressed in [f2b00e8](https://github.com/DataDog/lading/pull/1214/commits/f2b00e8cd96a22717d0ecc0862b2892c12f774bf)